src/ledger/LedgerManagerImpl.cpp

```diff
@@ -619,9 +624,23 @@ LedgerManagerImpl::startCatchup(CatchupConfiguration configuration,
         throw std::invalid_argument("Target ledger is not newer than LCL");
     }
 
-    setCatchupState(CatchupState::APPLYING_HISTORY);
-    assert(manualCatchup == mSyncingLedgers.empty());
+    if (manualCatchup)
+    {
+        if (getState() == LM_CATCHING_UP_STATE &&
```

Comment:
Will this if statement ever evaluate to `true`? It looks like command line already handles any catchup in progress: https://github.com/stellar/stellar-core/blob/master/src/main/CommandHandler.cpp#L554

Reply:
Good point, this probably means command handler catchup probably does not work (please do not confuse it with command line catchup).